lib/coverband/collectors/route_tracker.rb

```diff
@@ -13,7 +13,7 @@ class RouteTracker < AbstractTracker
       TITLE = "Routes"
 
       def initialize(options = {})
-        if Rails&.respond_to?(:version) && Gem::Version.new(Rails.version) >= Gem::Version.new("6.0.0") && Gem::Version.new(Rails.version) < Gem::Version.new("7.1.0")
+        if Rails&.respond_to?(:version) && Gem::Version.new(Rails.version) < Gem::Version.new("7.1.0")
           require_relative "../utils/rails6_ext"
```

Comment:
in this case we can delete this check and delete the rails6_ext file

Reply:
We still support rails 7.0, but the feature was added in 7.1 (https://github.com/rails/rails/pull/43755)